### ISSUE TITLE
Feature request: total red filter

### ISSUE THREAD

**Issue by strake** [GrapheneOS/os-issue-tracker#3718]
Just started using GrapheneOS and was disappointed at not being able to completely turn off green and blue pixels. I have seen friends do this on their devices. It is critical for visual astronomy to avoid green and blue light as it wrecks night vision.

It seems stock Android has some "color lens" option which does so. (I never used stock Android so i can't confirm this.)

**Comment by strake** [comment]
> You can set the filter much higher on GrapheneOS than the stock OS, and you shouldn't need more than the maximum. Did you try setting it to the maximum value?

Yes, and am seeing non-red light nonetheless. (i.e. can see multiple colors on the screen)

**Comment by dosutter** [comment]
> [@dosutter](https://github.com/dosutter) The Night Light feature in Display settings provides support for turning down blue light. GrapheneOS extends the range to where it can fully turn off blue pixels.

Oh I feel stupid now, I didn't know you could change the intensity of it when you clicked on this feature (and then you get a submenu). I thought using the toggle switch there was all that I could do. Thanks for getting back to me, and apologies.
